frontend/packages/ux-editor/src/types/FormContainer.ts

```diff
@@ -14,3 +16,19 @@ export interface FormContainer {
   edit?: KeyValuePairs;
   [id: string]: any;
 }
+
+export type ContainerComponent =
+  | ComponentType.Accordion
+  | ComponentType.AccordionGroup
+  | ComponentType.ButtonGroup
+  | ComponentType.Group;
```

Comment:
Foreslår å heller kalle denne `ContainerComponentType`, siden den ikke representerer en hel komponent, men bare typen. Er det for øvrig noen grunn til at du ikke har plassert den i en egen fil?

Hvis du lager et array med konteinertyper slik jeg nevnte i forrige kommentar, kan du for øvrig definere typen på denne måten:
`type ContainerType = typeof containerTypes[number]`